config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
huh wonder where you got this from hehe

this formatting was special built for our team, seems like you all have a different style guide so you'll definitely need to change these rules to fit your team. looks like you guys like curlies on separate lines and 4-space tabs